Source/WebCore/platform/graphics/GradientImage.cpp

```diff
@@ -86,7 +86,7 @@ void GradientImage::drawPattern(GraphicsContext& destContext, const FloatRect& d
         if (destContext.drawLuminanceMask())
             imageBuffer->convertToLuminanceMask();
 
-        m_cachedImage = ImageBuffer::sinkIntoNativeImage(WTFMove(imageBuffer));
+        m_cachedImage = WTFMove(imageBuffer);
```

Comment:
I think we need to remove the call to `ImageBuffer::sinkIntoNativeImage()` also from:
 1. `BitmapImage::drawLuminanceMaskPattern()`
 2. `SVGImage::drawPatternForContainer()`